docs/sources/tempo/operations/cross_tenant_query.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cross-tenant query federation
+menuTitle: Query
+description: Cross-tenant query federation
+weight: 70
+aliases:
+- /docs/tempo/operations/cross-tenant-query
+---
+
+
+# Cross-tenant query federation
+
+> NOTE: you need to enable `multitenancy_enabled: true` in the cluster for multi-tenant querying to work.
+> see [enable multi-tenancy]({{< relref "./multitenancy" >}}) for more details and implications of `multitenancy_enabled: true`.
+
+Tempo supports multi-tenant queries. where users can send list of tenants  multiple tenants.
```

Comment:
This sentence reads like it's not quite finished. What do you want to say here?

Reply:
reworded